src/framework/mpas_dmpar.F

```diff
@@ -5552,6 +5561,13 @@ subroutine mpas_dmpar_exch_halo_field2d_real(field, haloLayersIn)!{{{
               do while(associated(exchListPtr))
                 if(exchListPtr % endPointID == commListPtr % procID) then
                   do i = 1, exchListPtr % nList
+                    if (present(interiorCells)) then
```

Comment:
When running the tests, I did not see this print statement, which indicates no interior cells were updated via the normal halo transfer.